migrations/2_deploy_contracts.js

```diff
@@ -16,4 +17,11 @@ module.exports = (deployer, network) => {
   deployer.deploy([ColonyNetworkStaking]);
   deployer.deploy([EtherRouter]);
   deployer.deploy([Resolver]);
+
+  // We `require` the ReputationMiningCycle object to make sure
+  // it is injected in the `artifacts` variables during test
+  // preparation. We need this for the eth-gas-reporter.
+  // See https://github.com/cgewecke/eth-gas-reporter/issues/64
+  // This log prevents "variable not used" warnings.
+  console.log(ReputationMiningCycle);
```

Comment:
Can we disable "variable not used" here instead of logging the contract. That will be a very messy console output.

Reply:
Nice catch, we don't even need a variable... force of habit!